conjur/data_object/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+data_object package
+
+This package contain the data objects of the cli.
```

Comment:
This module contains the data objects of the client

Reply:
Fixed